model/common_datatypes.smithy

```diff
@@ -18,3 +18,18 @@ map UserDefinedValueMap{
 }
 
 document UserDefinedValue
+
+@enum([
+    {
+     value: "transient",
+     name: "transient"},
+    {
+     value: "persistent",
+     name: "persistent"}
```

Comment:
do we also need "defaults"?

Reply:
Thanks for pointing this out Sachet. Done in latest commit.